Dockerfile

```diff
@@ -1,18 +1,20 @@
-FROM alpine:3.17.2 as runtime
+FROM alpine:latest AS runtime
 
-ENV APPHOME /opt/readalong-studio
+ENV APPHOME=/opt/readalong-studio
 
 # Lean, optimized installation of system dependencies
 RUN apk add python3 py3-numpy py3-yaml git ffmpeg
 
-FROM runtime as build
+FROM runtime AS build
 WORKDIR $APPHOME
 RUN apk add python3-dev py3-pip gcc g++ musl-dev ninja
 RUN python3 -m venv --system-site-packages $APPHOME/venv
 RUN git clone https://github.com/roedoejet/g2p.git
 COPY requirements*.txt $APPHOME/
 RUN . $APPHOME/venv/bin/activate \
     && python3 -m pip install -r $APPHOME/requirements.txt
+RUN . $APPHOME/venv/bin/activate \
+    && python3 -m pip install soundswallower
```

Comment:
it wasn't in the requirements.txt?

Reply:
It's not, because heroku doesn't do alignment and therefore it does not need soundswallower. It's only in pyproject.toml.

Reply:
But I want to install it early to optimize docker caching effects.

Reply:
ah ok!